osutil/mkfs/mkfs.go

```diff
@@ -111,6 +111,7 @@ func mkfsExt4(img, label, contentsRootDir string, deviceSize, sectorSize quantit
 		// no need to fake it if we're already root
 		cmd = exec.Command(mkfsArgs[0], mkfsArgs[1:]...)
 	}
+	cmd.Env = os.Environ()
```

Comment:
I don't see how this can work: `cmd.Env` is already set to the current environment (see [the documentation here](https://pkg.go.dev/os/exec@go1.13.15#Cmd)), so this line should not have any effect.

Reply:
hmm perhaps I misread the docs originally. Let me double check some things and get back to this.